projects/sandbox/vizapp/vizapp/distributions.py

```diff
@@ -77,8 +77,8 @@ def get_foreground(
 
         with h5py.File(shift / "injection" / "params.h5") as f:
             event_times = f["geocent_time"][:]
-            h1_snrs = f["H1_snr"][:]
-            l1_snrs = f["L1_snr"][:]
+            h1_snrs = np.sqrt(f["H1_snr"][:])

```

Comment:
Really quick: given that we're now using ML4GW with @deepchatterjeeligo 's fixes in it, this square rooting should actually be incorrect

Reply:
Done